### PR TITLE
added Chia portfolio to engineers folder

### PR DESCRIPTION
**Pull request by chlin501**

### CODE REVIEW

File: src/engineering.html [hunk 1]
```diff
@@ -10,323 +10,449 @@
...
-  </head>
-  <body>
-    <header>
-      <h1 class="desktop">
+</head>
+<body>
+<header>
+    <h1 class="desktop">
         508.dev
-      </h1>
```
Comment (komali2) [inline src/engineering.html]: reissue without these indentation changes, please

**Review by komali2 — CHANGES_REQUESTED** [review]
looks like your editor made a ton of indentation changes, can you reissue without them?

File: src/engineering.html (hunk 1 above)
Comment (chlin501) [reply src/engineering.html]: Updated. Please let me know if any further concerns. 

By the way, I notice there is a div tag missing (no closed &lt;/div&gt;  error). So I reformat the code.

Comment (komali2) [reply src/engineering.html]: oh, was the indentation stuf from that? well, good find if so, can you issue a separate PR about that though?

**Review by komali2 — APPROVED** [review]

**Pull request merged**